bin/rails

```diff
@@ -3,12 +3,12 @@
 # installed from the root of your application.
 
 ENGINE_ROOT = File.expand_path("..", __dir__)
-ENGINE_PATH = File.expand_path("../lib/dato/rails/engine", __dir__)
-APP_PATH = File.expand_path("../test/dummy/config/application", __dir__)
+ENGINE_PATH = File.expand_path("../lib/dato/engine", __dir__)
+APP_PATH = File.expand_path("../spec/dummy/config/application", __dir__)
 
 # Set up gems listed in the Gemfile.
 ENV["BUNDLE_GEMFILE"] ||= File.expand_path("../Gemfile", __dir__)
 require "bundler/setup" if File.exist?(ENV["BUNDLE_GEMFILE"])
 
-require "rails/all"
+require "rails"
```

Comment:
i needed to change this; otherwise it always asked me for a DB when calling `rails console`

Reply:
yet `rails s` does not run, with this changes i get at least another error

Reply:
Ok!